### PR TITLE
Add check for non-inclusive language

### PR DESCRIPTION
**Pull request by nhosoi**
Add a check for usage of terms and language that is considered non-inclusive. We are using the woke tool for this with a wordlist that can be found at https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/config_files/woke.yml

CHANGELOG.md - cleanup non-inclusive words.

### CODE REVIEW

**Comment by Jakuje** [comment]
Would be great if the action (or at least the commit message or PR description) would have some human-readable description what does the CI action does and why.

File: .github/workflows/woke.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# yamllint disable rule:line-length
+# name: woke
```
Comment (richm) [inline .github/workflows/woke.yml:2]:
```suggestion
name: woke
```

**Comment by richm** [comment]
> Would be great if the action (or at least the commit message or PR description) would have some human-readable description what does the CI action does and why.

Sorry - we get so caught up in the tasks that the core team is focused on, we forget that other people might not realize what we're doing.

@nhosoi We should update all of the existing PRs like this:

PR title: Add check for non-inclusive language

PR body: Add a check for usage of terms and language that is considered non-inclusive.  We are using the `woke` tool for this with a wordlist that can be found at https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/config_files/woke.yml

in .github/workflows/woke.yml at the top-level it should have
`name: Check for non-inclusive language`

Then the PR status will look like `Check for non-inclusive language / woke (pull_request)`

@Jakuje is this sufficient?

**Comment by nhosoi** [comment]
Thank you for your suggestions, @richm. I think I updated the PR following your advice.

Sorry, @Jakuje, for giving you this surprise... Does this updated PR look better?

**Comment by nhosoi** [comment]
@richm, I put `[WIP]` to the PR title because once https://github.com/get-woke/woke/pull/252 (or its alternative) is merged, we may need to add a new option to the action to allow the warning-level words. Do you think it's ok to wait for the woke PR's fate? Or should we handle the woke PR separately?

**Comment by Jakuje** [comment]
Thanks for the improvements. Looks better. I have only concern about the modification of the changelog how it is done now.

**Comment by richm** [comment]
> Thanks for the improvements. Looks better. I have only concern about the modification of the changelog how it is done now.

Not sure what you mean - what's your concern about the changelog?

**Comment by richm** [comment]
> > Thanks for the improvements. Looks better. I have only concern about the modification of the changelog how it is done now.
> 
> Not sure what you mean - what's your concern about the changelog?

Is it that the intention of the changelog is that it is readable by end users, and having lines like
```
- update to tox-lsr 2.4.0 - add support for ansible-test sanity with docker // wokeignore:rule=sanity
```
is going to be distracting? 
We could tell `woke` to skip `CHANGELOG.md` completely - but then we run the risk that some sort of non-inclusive language getting into CHANGELOG.md, which would be even worse than having it in other code which isn't usually intended to be read.
We could edit `CHANGELOG.md` to change or remove those instances of non-inclusive language - this may be the best option.

**Comment by Jakuje** [comment]
yes, the linter hint might be distracting or confusing to users. I think editing the changelog not to contain potentially non-inclusive language should be preferred way (this is why we do it anyway, aren't we).

**Comment by nhosoi** [comment]
[citest]

**Comment by nhosoi** [comment]
[citest bad]

**Review by Jakuje — APPROVED** [review]

**Comment by nhosoi** [comment]
Thanks for your approval, @Jakuje! Since I have no privilege to do so, if you could merge this pr, I'd appreciate it. Thanks!

**Review by t184256 — APPROVED** [review]
Looks fine to me, thanks.

**Pull request merged**